setup.py

```diff
@@ -46,7 +46,7 @@
 ]
 
 requirements = [
-    "aicsimageio>=3.1.2",
+    "aicsimageio>=4.1.0",
```

Comment:
If you want TwoDWriter / reader you should make this `aicsimageio[base-imageio]` or `aicsimageio[all]`

Reply:
thanks!!  will fix that